src/VisualStudio.Roslyn.SDK/Roslyn.SDK/CSharpDiagnostic.vstemplate

```diff
@@ -4,7 +4,7 @@
             xmlns="http://schemas.microsoft.com/developer/vstemplate/2005">
   <TemplateData>
     <Name>Analyzer with Code Fix (.NET Standard)</Name>
-    <Description>Create a C# analyzer that comes with a code fix and generates diagnostics. The analyzer can be deployed as either a NuGet package or a VSIX extension.</Description>
+    <Description>Create a C# analyzer that comes with a code fix and generates diagnostics. The analyzer can be deployed as a NuGet package.</Description>
```

Comment:
💭  I actually like this wording change, even if we don't remove the VSIX project.